components/user-modal/content/step1.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Radio from 'components/forms/radio';
+
+import { userTypeOptions, useTypeOptions } from './constants';
+
+const Step1 = ({ onClick, userData, handleUserData, user }) => (
+  <section>
+    <h4 className="mb-4">Can you tell us a bit about yourself?</h4>
+    <h5 className="mt-4">
+      In which sector do you work: * (pick the one that brought you to this map)
+    </h5>
+    <div className="form-group user-modal-radio-input-container">
+      {userTypeOptions.map(option => (
+        <>
+          <Radio
+            key={option.slug}
+            id={`job-role-${option.slug}`}
+            className="mr-4"
+            name="job-role"
+            checked={option.slug === (userData.job_role || userData.job_role_description)}
+            onChange={() => handleUserData('job_role', option.slug)}
+          >
+            {option.label}
+          </Radio>
+          {option.slug === 'other' && (
+            <>
+              <input
+                type="text"
+                id="job_role_description"
+                name="job_role_description"
+                className="user-modal-text-input"
+                placeholder="insert your profession"
+                disabled={option.slug !== 'other'}
+                value={userData.job_role_description}
+                onChange={({ currentTarget }) =>
+                  handleUserData('job_role_description', currentTarget.value)
+                }
+                required={userData.job_role_description === 'other'}
+              />
+              <label aria-label="hidden" className="visually-hidden">
+                job role description
+              </label>
```

Comment:
Please delete this, this is horrifying 😭😭😭.

If you don't use a visible `<label />`, you can use `aria-label` on the input itself, as I've suggested above. If you use one, then the label and the input must be connected in one of two ways:
- The `<label />` contains the input
- The `<label />` has a `for` (in JSX: `htmlFor`) attributes that references the input's ID

Here not only the label is not connected to the input (so it's useless), but because you put `aria-label="hidden"`, the screen reader users will actually hear “hidden” instead of “job role description”.

If you are unsure about aria attributes, don't use them. It's worse to use them incorrectly than not using them. And obviously, it's even better if the input has a visible label.